android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/GenericPopupDialog.java

```diff
@@ -90,8 +91,9 @@ public void onCreate(Bundle savedInstanceState) {
 
         // support translation of sub-forms
         Intent activityIntent = getActivity().getIntent();
-        translateSubForm =  activityIntent == null ? translateSubForm
-                : activityIntent.getBooleanExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION, false);
+        translateSubForm = activityIntent.hasExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION) ?
```

Comment:
the check for ` activityIntent == null` has been removed, is that intentional?

Reply:
https://github.com/OpenSRP/opensrp-client-native-form/pull/529/files#r535217012

Reply:
Check Added